annotations.md

```diff
@@ -28,7 +28,16 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.revision** Source control revision identifier for the packaged software.
 * **org.opencontainers.image.vendor** Name of the distributing entity, organization or individual.
 * **org.opencontainers.image.licenses** License(s) under which contained software is distributed as a [SPDX License Expression][spdx-license-expression].
-* **org.opencontainers.image.ref.name** Name of the reference for a target (string). SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+* **org.opencontainers.image.ref.name** Name of the reference for a target (string).
+  * SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+  * Character set of the value SHOULD conform to alphanum of `A-Za-z0-9` and separator set of `-._:@/+`
+  * An EBNF + regular expression like:
```

Comment:
Why use EBNF (presumably [ISO/IEC 14977:1996][1]) instead of ABNF ([RFC 5234][2])?  ABNF is much more available than EBNF's [restrictively-licensed zipped PDF][3].  ABNF covers the same bits you need here ([`=` for rule definition][4] and [`/` for alternates][5]).  For folks who prefer to read EBNF, that spec defines `/` as an alternative definition-separator-symbol (with `|` being the canonical EBNF definition-separator-symbol).

[1]: https://www.iso.org/standard/26153.html
[2]: https://tools.ietf.org/html/rfc5234
[3]: http://standards.iso.org/ittf/PubliclyAvailableStandards/s026153_ISO_IEC_14977_1996%28E%29.zip
[4]: https://tools.ietf.org/html/rfc5234#section-2.2
[5]: https://tools.ietf.org/html/rfc5234#section-3.2